content/docs/atom-guide/dev-guide/render_pipelines.md

```diff
@@ -0,0 +1,85 @@
+| Subsurface scattering | Calculates subsurface scattering for materials (such as skin) by using a screen space technique. | Yes | No |
+| Deferred fog | Applies a height-based fog to the scene. | Yes | No |
+| Screen space reflections (SSR) | Implements SSR by using renderered lighting buffer to approximate real-time reflections. | Yes | No |
+| Depth of field | Applies a bokeh-shaped depth of field effect to the scene that's based on the scene's depth and camera focus paramters. | Yes | No |
+| Bloom | Applies a bloom effect to the rendered image, making bright pixels bleed out into neighboring pixels. | Yes | No |
+| Subpixel morphological anti-aliasing (SMAA) | Applies SMAA as an anti-aliasing technique. | Yes | No |
+| Temporal anti-aliasing (TAA) | TAA uses motion vectors and temporal super-sampling to reduce aliasing in the final image. | Yes | No |
+| Light adaptation | Adjusts brightness of the final image based on the scene's average luminance, so the image appears neither too dark nor too bright. | Yes | Yes |
+| Look modification and color grading | Allows artists to adjust the final look of the image by using look up tables (LUTs). | Yes | Yes |
+| Display mapping | Adjusts the color values of the final image based on the color range that the screen expects. | Yes | Yes |
+| User interface (UI) | Renders 2D UI components. | Yes | Yes |
+| Auxilary geometry (AuxGeom) | Renders various geometry and text for in-editor tools and debugging, such as gizmos, grids, debug lines, and shapes. | Yes | Yes |
+
+
+
+## File structure
+
+A render pipeline is stored as an `.azasset` file, which is a JSON file that links to the root pass of the render pipeline and contains other configurations. In this file, the `RootPassTemplate` parameter contains the name of pipeline's root pass.
+
+The following example is the `MainRenderPipeline.azasset` file. The root pass template is called `MainPipeline`, which links to the `MainPipeline.pass` file. 
+```json
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "RenderPipelineDescriptor",
+    "ClassData": {
+        "Name": "MainPipeline",
+        "MainViewTag": "MainCamera",
+        "RootPassTemplate": "MainPipeline",
+        "RenderSettings": {
+            "MultisampleState": {
+                "samples": 4
+            }
+        }
+    }
+}
+
+```
+
+
+You can find the following render pipeline files in your O3DE engine source:
+- **Main Rendering Pipeline**: `Gems\Atom\Feature\Common\Assets\Passes\MainRenderPipeline.azasset`
+- **Low-end Rendering Pipeline**: `Gems\Atom\Feature\Common\Assets\Passes\LowEndRenderPipeline.azasset`
+
+
+## Modifying the render pipeline
+
+Since the render pipeline is driven by modular passes, it's easy to add or remove passes and update connections. You can modify the main render pipeline, or create a custom render pipeline for your team. You may want to use a custom render pipeline for a number of reasons, such as:
```

Comment:
```suggestion
Since modular passes drive the render pipeline, it's easy to add or remove passes and update connections. You can modify the main render pipeline or create a custom render pipeline for your team. You might want to use a custom render pipeline for a number of reasons, such as:
```
* In the first sentence:
  * Revised for active voice.
  * Consider avoiding calling this "easy", which can be condescending to a reader.